data/README.rst

```diff
@@ -0,0 +1,107 @@
+Software Directory README
```

Comment:
Do you have concerns except the readability on GitHub and in plaintext (both of which is fine with rst in my opinion)?